pool/pool.go

```diff
@@ -185,28 +536,30 @@ type clientPack struct {
 	address string
 }
 
-type prmCommon struct {
+type prmContext struct {
```

Comment:
Can you pls explain why did we highlight `prmContext`?

Reply:
Since we have `clientWrapper` that accepts object parameters with `prmCommon` and doesn't use fields `defaultSession`, `verb`, `addr` (these fields are used only in `pool` methods) I decided highlight `prmContext`.

Reply:
I'd then name it `prmObjectSessionContext` since `prmContext` seems too abstract (or you can leave your name and write some explanation in type docs).